### DISCUSSION TITLE
Weird behaviour after rebuilding

### DISCUSSION THREAD

**Discussion by MelleNooijen** [vgmtrans/vgmtrans#199] [Q&A]
I've just built the new version from the `master` branch, but when I try to launch it I get the following error:
```
QIODevice::read (QFile, ":/qdarkstyle/style.qss"): device not open
```
And the program doesn't look right at all, and doesn't function (see [this](http://mellemws.my.to/direct/Screenshot_from_2021-03-14_17-33-41.png))
Is this how it's supposed to be due to the experimental nature of the `master` branch, or did I do something wrong while building?

**Comment by sykhro** [comment]
Yes, `master` still doesn't contain any of the Qt improvements, they will be backported sometime soon. If you need a functional Qt version, compile the `refactor` branch.
Right now all you can don in `master` is drag-and-drop files and listen to them by playing space

**Reply by MelleNooijen** [reply]
Thanks for your fast response, I didn't know `master` was so experimental :sweat_smile:

**Reply by sykhro** [reply]
`master` wasn't touched in ages, it took a while of experimenting to figure out a decent plan to follow. There's been more contributions recently, I expect the Qt stuff to be backported before this summer

**Comment by sykhro** [comment]
Try building `master` now and you should see the difference :)
